internal/chart/procfile.go

```diff
@@ -52,8 +52,7 @@ func ParseProcfile(content string) (*Procfile, error) {
 	for _, process := range procfile {
 		if p := procfileRegex.FindStringSubmatch(process); p != nil {
 			name := p[1]
-			cmd := p[2]
-			processes[name] = []string{strings.TrimSpace(cmd)}
+			processes[name] = []string{strings.TrimSpace(name)}
```

Comment:
This change seems to work, but even with the PR description, I'm still confused as to why we can disregard the Processes' Commands here.

Reply:
I'll add an additional comment to clear things up, since it isn't particularly intuitive. The Procfile that pack wants looks exactly like you would expect, something like
```
web: python app.py
worker: python app.py
```
The thing is, in the image that pack creates things look a bit different. The names specified in the Procfile end up becoming executables located under `/cnb/process`, so in this case we have `web` and `worker` within that folder. If I want to execute the commands specified in the Procfile I have to run them with either `web` or `worker` since they are in the path whereas something like python won't be.

Reply:
if it's deployment from source, a process' command line has to match `name`. 
For deployment from an image, we provide the right command line
```
Application: myapp
Framework: dev
Address: http://myapp.10.98.0.185.shipa.cloud

No environment variables.
DEPLOYMENT VERSION    IMAGE                                        PROCESS NAME    WEIGHT    STATE        CMD
1                     docker.io/shipasoftware/bulletinboard:1.0    web             100%      1 running    docker-entrypoint.sh npm start

```
so it looks OK to me